commands/report.py

```diff
@@ -221,17 +221,17 @@ def report(accounts, config, args):
         },
     ]
 
-    for account in accounts:
-        account = Account(None, account)
-        print("  - {}".format(account.name))
+    # for account in accounts:
```

Comment:
I can't merge this due these being commented out. Maybe a command-line flag could avoid these checks?